src/transport.js

```diff
@@ -35,6 +35,17 @@ module.exports = env => {
           domain: env('MAILGUN_DOMAIN')
         }
       }))
+    case 'smtp':
+      return nodemailer.createTransport({
+        host: env('SMTP_HOST'),
+        port: env('SMTP_PORT'),
```

Comment:
Doesn't this require port to be a number? Maybe not, just asking.

Reply:
The parameter is the port to connect to (defaults to 587 is secure is false or 465 if true). This parameter would be added too.